proto/account/api_key.proto

```diff
@@ -23,6 +23,8 @@ message APIKey {
     UNKNOWN = 0;
     SDK_CLIENT = 1;
     SDK_SERVER = 2;
+    API_READ_ONLY = 3;
+    API_READ_WRITE = 4;
```

Comment:
I think we should have at least 3 roles for public APIs, just like the account roles.
How about this?

```suggestion
    PUBLIC_API_READ_ONLY = 3;
    PUBLIC_API_WRITE = 4;
    PUBLIC_API_ADMIN = 5; // For sensitive data
```

or

```suggestion
    OPEN_API_READ_ONLY = 3;
    OPEN_API_WRITE = 4;
    OPEN_API_ADMIN = 5; // For sensitive data
```

Reply:
I think that utilizing `OPEN_API_` would be more appropriate. 

Many open-source projects that support REST API access adhere to the [OpenAPI Specification](https://swagger.io/specification/).

Reply:
I prefer just `API_XXX` in these three options because it already has the same sense as the other two.
OpenAPI explains how we implement the API. We need to know what implementation we use. It could be gRPC or another. It's unlikely, but it could also be possible that we will add an implementation of GraphQL to meet the user's demands.

Reply:
Only API for me it's abstract.
It could be anything.
Server API, SDK API, Client API.
If the implementation it's not decided, I still prefer PUBLIC as prefix because it's not related to the implementation.
Or if you guys have a better idea.

Reply:
OK, let's go with `PUBLIC_API_` 🚀